packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/breadcrumbs.twig

```diff
@@ -1,14 +1,23 @@
-
-<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
-  <nav class="ma__breadcrumbs__container">
-    <div class="ma__breadcrumbs__label">
-      More in:
-    </div>
-    <ol class="ma__breadcrumbs__items">
-      <li class="ma__breadcrumbs__item"><a href="/">Home</a></li>
-      {% for link in breadcrumbs.links %}
-        <li class="ma__breadcrumbs__item"><a href="{{link.href}}">{{link.text}}</a></li>
-      {% endfor %}
-    </ol>
-  </nav>
-</section>
+<section class="ma__breadcrumbs {{breadcrumbs.light ? 'ma__breadcrumbs--light' : ''}}">
```

Comment:
what does the light property do? How is `ma__breadcrumbs--light` from the regular one?

Reply:
Honestly this was part of the original integration, the light variation has a white background which is what we need.

Reply:
I see! In this case, can you just modify the documentation to note the difference when setting "light" to true? I was confused by what it was supposed to do